src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenInParametersTests.cs

```diff
@@ -1818,5 +1813,96 @@ static void Main()
     }
 }", expectedOutput: "5-9");
         }
+
+        [Fact]
+        [WorkItem(530136, "https://devdiv.visualstudio.com/DevDiv/_workitems?id=530136")]
+        public void OperatorsWithInParametersFromMetadata_Binary()
+        {
+            var reference = CreateStandardCompilation(@"
+public class Test
+{
+    public int Value { get; set; }
+
+    public static int operator +(in Test a, in Test b)
```

Comment:
>public static int operator +(in Test a, in Test b) [](start = 4, length = 50)

Please add a test for scenarios when ```in``` is only on the left and ```in``` is only on the right.